server/src/test/java/org/apache/druid/metadata/SQLMetadataStorageActionHandlerTest.java

```diff
@@ -257,10 +237,13 @@ public void testDuplicateInsertThrowsEntryExistsException()
     Map<String, Object> status = ImmutableMap.of("count", 42);
```

Comment:
This test name needs updating as the exception no longer exists: `testDuplicateInsertThrowsEntryExistsException`